src/System.Management.Automation/engine/parser/SemanticChecks.cs

```diff
@@ -248,9 +246,7 @@ public override AstVisitAction VisitAttribute(AttributeAst attributeAst)
 
                     if (!namedArg.ExpressionOmitted && !IsValidAttributeArgument(namedArg.Argument, constantValueVisitor))
                     {
-                        string errorId;
-                        string errorMsg;
-                        GetNonConstantAttributeArgErrorExpr(constantValueVisitor, out errorId, out errorMsg);
+                        (string errorId, string errorMsg) = GetNonConstantAttributeArgErrorExpr(constantValueVisitor);
```

Comment:
No need to assign like this, just do `var error = ..` and then `_parser.ReportError(namedArg.Argument.Extent, error.id, error.msg)`